### PR TITLE
Portfolio-Mobile finish

### PR DESCRIPTION
**Pull request by Simiran777**
# Overview
**Adding the HTML file to display the portfolio page
**Adding the CSS file for the styling of the portfolio page
** Fixed the errors using linters

# Why is the update required?
**This project helps in understanding the use of linters, Git & Github and Github flow and documenting the repo in a professional way. Thus, it should include error-free linter-checked files to display the functionality of linters.
** I updated the HTML and CSS files to make them more detailed and interactive.
**The design is made Microverse Students Figma Template for creative designing of the webpage.

# What changes does this update make?
** The code runs at the beginning to display a page of my portfolio.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -149,6 +147,22 @@ <h3 class="skills_list">Skills<p><i class="arrow right"></i></p>
...
+                built or a project that needs coding. I’d love to help with it.</p>
+            <form action="https://formspree.io/f/xeqdkkpj" method="POST">
+                <ul class="form_fields">
+                    <li><input class="name" name="for-name" type="text" placeholder="Your Name" maxlength="30" required>
+                    </li>
+                    <li><input class="email" name="for-email" type="email" placeholder="Your Email" required></li>
+                    <li><textarea class="message" name="for-msg" id="Msg" cols="30" rows="8"
+                            placeholder="Write your message here" maxlength="500" required></textarea></li>
+                </ul>
+                <button class="button" id="btn" type="button">Get In Touch</button>
```
Comment (levy002) [inline index.html]: - [ ] Kindly make sure the role of your button is to submit the filled information. While the type of your button is `button` we can't be able to submit the form. I highly recommend changing the button type to `submit` after that you will be able to submit your form correctly. :wink:

File: style.css [hunk 2]
```diff
@@ -367,3 +367,93 @@ h3 {
   transform: rotate(45deg);
   -webkit-transform: rotate(45deg);
 }
+
+.contact-form {
+  background-color: #fff;
+}
+
+.Contact_form {
+  background-image: url(./images/contact_form.png);
```
Comment (levy002) [inline style.css:376]: - [ ] Kindly make sure in your `images` folder you have this `contact_form.png` image. At this moment, our website can't find this particular image. Kindly make sure to add it as well.

    - ### Your design
    image

    - ### Figma design

   image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @Simiran987 @Amrit131   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Simiran777) [reply index.html]: fixed

File: style.css (hunk 2 above)
Comment (Simiran777) [reply style.css:376]: Added

**Comment by Simiran777** [comment]
@levy002 I have made the required changes, please kindly check it.

**Comment by levy002** [comment]
Hello @Simiran987. Thank you for implementing all suggested changes. Kindly go ahead and submit for another review 😉

**Review by victorokoroji — APPROVED** [review]
Hi @Simiran987, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Good UI✅ 
- Excellent communication with code reviewer ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**